src/types/clipboard.ts

```diff
@@ -10,7 +10,7 @@ export type ClipboardContent = { [type in ClipboardMIMEType]?: string };
 export interface ClipboardOptions {
   pasteOption?: ClipboardPasteOptions;
   selectTarget?: boolean;
-  isCutOperation?: boolean;
+  isCutOperation: boolean;
```

Comment:
nitpick, we could move this at the top of the interface declaration since it's mandatory